x-pack/plugins/security/server/user_profile/user_profile_service.ts

```diff
@@ -30,6 +38,18 @@ const MIN_SUGGESTIONS_FOR_PRIVILEGES_CHECK = 10;
  * A set of methods to work with Kibana user profiles.
  */
 export interface UserProfileServiceStart {
+  /**
+   * Retrieves a user profile for the current user extracted from the specified request. If the profile isn't available,
+   * e.g. for the anonymous users or users authenticated via authenticating proxies, the `null` value is returned.
+   * @param params Get current user profile operation parameters.
+   * @param params.request User request instance to get user profile for.
+   * @param params.dataPath By default Elasticsearch returns user information, but does not return any user data. The
+   * optional "dataPath" parameter can be used to return personal data for the requested user profiles.e.
+   */
+  getCurrent<D extends UserProfileData, L extends UserProfileLabels>(
+    params: UserProfileGetCurrentParams
+  ): Promise<UserProfileWithSecurity<D, L> | null>;
```

Comment:
__note:__ Unlike other publicly exposed methods returning `UserProfileWithSecurity` from the `getCurrent` is save and in line with what we do for the `authc.getCurrentUser`.